src/main/java/fact/utils/ElementwiseMultiplyDoubleArray.java

```diff
@@ -0,0 +1,46 @@
+package fact.utils;
+
+import stream.Data;
+import stream.Processor;
+import stream.annotations.Parameter;
+import fact.features.singlePulse.timeSeriesExtraction.ElementWise;
+
+public class ElementwiseMultiplyDoubleArray implements Processor {
+
+    @Parameter(
+        required = true,
+        description = "Key to your double array."
+    )
+    private String inputKey;
+
+    @Parameter(
+        required = true,
+        description = "Key to the output integer array."
```

Comment:
Also: Does `Elementwise` create a copy? If not, the input array is `overwritten` in the sense that the operation happens inplace as the data item contains a reference to the array, which got multiplied by 5.